.github/workflows/release.yml

```diff
@@ -34,12 +34,7 @@ jobs:
         run: |
           PLUGIN_NAME=$(grep "^PLUGIN_NAME=" gradle.properties | cut -d '=' -f 2)
           VERSION=$(grep "^VERSION_NAME=" gradle.properties | cut -d '=' -f 2)
-
-          if [[ $VERSION == *-SNAPSHOT ]]; then
-            RELEASE_NAME="Nightly"
-          else
-            RELEASE_NAME=$PLUGIN_NAME-$VERSION
-          fi
+          RELEASE_NAME=$PLUGIN_NAME-$VERSION
```

Comment:
And I believe we should remove `Valkyrie` prefix for tags, would be clear enough.

Reply:
Will the old tags have the old prefix? 
as for me prefix not critical)

Reply:
Yeah, old tags will keep their names, new tags will be like `0.6.0-SNAPSHOT`.

Reply:
If it is really necessary, I don't mind changing it 🙂

Reply:
The version name for the tag is very clear here, we don't mix more than one project in this repository.

Reply:
9a08ff5a169f0d7eff46446b8236b6eba2f37a4c

Reply:
Seems like we need to manually delete the 'Nightly' release the first time, but I could be mistaken.